docs/getting-started/installation.md

```diff
@@ -0,0 +1,51 @@
+# How to install propsd #
+
+[Releases of propsd][releases] include both source tarballs and Debian
+packages. Debian and Ubuntu based Linux distributions can use the pre-built
```

Comment:
Is it worth linking to latest (https://github.com/rapid7/propsd/releases/latest)?

Curious if users would just click through things during the reading of the installation guide.